packages/playground/blueprints/src/lib/compile.ts

```diff
@@ -170,7 +172,10 @@ export function compileBlueprint(
 	const wpCliStepIndex = blueprint.steps?.findIndex(
 		(step) => typeof step === 'object' && step?.step === 'wp-cli'
 	);
-	if (wpCliStepIndex !== undefined && wpCliStepIndex > -1) {
+	if (
+		blueprint?.features?.wpCli === true ||
+		(wpCliStepIndex !== undefined && wpCliStepIndex > -1)
+	) {
```

Comment:
The warning [here](https://github.com/WordPress/wordpress-playground/pull/1629/files#diff-111618e10ba828ef4596cfba7f3a377a0c7cefc4b2432d96a13a9bd77f462987R185) says "The wpCli step used in your Blueprint" which could be confusing because someone can enable the "wpCli" feature without including a wpCli step.